README.md

```diff
@@ -19,8 +19,13 @@ IEEE Transactions on Audio, Speech and Language Processing, In press, 2018.
 Installation
 ------------
 
+To install the keras-based implementation:
 ```
-pip install autopool
+python -m pip install autopool[keras]
+```
+For the tensorflow implementation:
+```
+python -m pip install autopool[tf]
```

Comment:
What happens if you call `pip install autopool` withtout the keras/tf specification? Is there a default behavior?

Reply:
It installs with no dependencies.  We actually have no (runtime) dependencies beyond the python core until getting into specific framework implementations.